docs/faq.md

```diff
@@ -16,12 +16,12 @@ If you don’t see your question here, feel free to drop by `#docker-compose` on
 freenode IRC and ask the community.
 
 
-## Can I control service startup order?
+### Can I control service startup order?
```

Comment:
So this is to deal with a theme issue, and making this change will likely cause problems when we change the docs css :/